apps/console/src/features/identity-providers/components/settings/attribute-management/attribute-mapping-list-item.tsx

```diff
@@ -0,0 +1,301 @@
+    //                 <Grid.Column width={ editingMode ? 7 : 8 }>
+    //                     <Field.Input
+    //                         required
+    //                         label={ !editingMode && "External IdP Attribute" }
+    //                         placeholder="Enter external IdP attribute"
+    //                         ariaLabel="External IdP Attribute Mapping Value"
+    //                         inputType="default"
+    //                         maxLength={ 120 }
+    //                         minLength={ 1 }
+    //                         width={ 16 }
+    //                         value={ initialValues?.mappedValue }
+    //                         listen={ (value) => setMappedValue(String(value)) }
+    //                         name="mappedValue"/>
+    //                 </Grid.Column>
+    //                 <Grid.Column width={ 8 }>
+    //                     <Field.Dropdown
+    //                         search clearable required
+    //                         label={ !editingMode && "Maps to" }
+    //                         width={ 16 }
+    //                         options={ _getListOfAvailableAttributes() }
+    //                         placeholder="Select mapping attribute"
+    //                         listen={ (value) => setLocalClaimId(String(value)) }
+    //                         value={ initialValues?.localClaimId }
+    //                         noResultsMessage="Try another attribute search."
+    //                         ariaLabel="Local Claim Attribute"
+    //                         name="localClaimId"/>
+    //                 </Grid.Column>
+    //                 { editingMode && (
+    //                     <Grid.Column width={ 1 }>
+    //                         <Field.Button
+    //                             disabled={ !mappedValue || !localClaimId }
+    //                             icon="checkmark"
+    //                             type="submit"
+    //                             name="submit-button"
+    //                             ariaLabel="Attribute Selection Form Submit Button"
+    //                             buttonType="secondary_btn"/>
+    //                     </Grid.Column>
+    //                 ) }
+    //             </Grid.Row>
+    //             { !editingMode && <Grid.Row columns={ 1 }>
+    //                 <Grid.Column width={ 16 } textAlign="right">
+    //                     <Field.Button
+    //                         disabled={ !mappedValue || !localClaimId }
+    //                         type="submit"
+    //                         name="submit-button"
+    //                         label="Add Attribute Mapping"
+    //                         ariaLabel="Attribute Selection Form Submit Button"
+    //                         buttonType="primary_btn"/>
+    //                 </Grid.Column>
+    //             </Grid.Row> }
+    //         </Grid>
+    //     </Form>
+    // );
```

Comment:
Remove commented code if unnecessary. Please check other similar occurances.

Reply:
Removed comment in b49a285